example/rds-cluster-tf/variables.tf

```diff
@@ -103,7 +113,7 @@ variable "lambda_timeouts" {
 variable "parameter_group_family" {
   description = "Database family to use for creating database parameter group"
   type = string
-  default = "aurora-postgresql11"
+  default = "aurora-postgresql13"
```

Comment:
I think we have a separate `parameter_group_family_v13` below? This change updates the default?

Reply:
Yes - I believe this change is incorrect, this is an artifact of a mediocre upgrade path.  This release should *remove* this parameter group, but once that is done will make it such that users coming from releases prior to the PG11->PG13 update cannot upgrade without hitting a version with this first. 

If you're curious what I mean, take a look at the release notes for the PG upgrade and/or we can tag up on this.

Reply:
I agree that we no longer need "parameter_group_family" and "parameter_group_family_v13". Do you recommend removing "parameter_group_family" and leaving "parameter_group_family_v13" as part of this PR?